lib/assets.js

```diff
@@ -102,17 +112,27 @@ Assets.prototype.serveAsset = function (req, res, next) {
 
   var contentType = asset.contentType;
 
-  if (contentType.match(/^text\/|\/json$|\/javascript$/)) {
-    contentType += "; charset=UTF-8";
-  }
+  if (contentType) {
+    if (contentType.match(/^text\/|\/json$|\/javascript$/)) {
+      contentType += "; charset=UTF-8";
+    }
 
-  res.setHeader("Content-Type", contentType);
+    res.setHeader("Content-Type", contentType);
+  }
   res.setHeader("Content-Length", asset.length);
 
   if (req.method === "HEAD") {
     return res.end();
   }
 
+  if (!asset.buffer) {
+    var localFilePath = pathModule.join(this.options.buildDir, assetName);
+    return fs.readFile(localFilePath, function (err, data) {
+      if (err) return next(err);
+      res.end(data);
+    });
+  }
+
   res.end(asset.buffer);
```

Comment:
Oh, scratch that — I see you’ve added a `return` on 130. I’ll put the else here anyway so nobody else gets tripped up! :nail_care: